src/components/box/box.stories.mdx

```diff
@@ -119,6 +119,94 @@ Please note that in order to see the `position: fixed` Box, you will need to vie
   </Story>
 </Canvas>
 
+### Gap
+
+The `Box` component also supports the `gap` property when its `display` property is set to either 
+`flex` or `inline-flex`. See the [prop table](#props) below for more information on the types it accepts
+
+<Canvas>
```

Comment:
in might be worth demonstrating the 2-value syntax (eg `gap="10px 20px"`) as part of this story?

Reply:
I'll add a demo of the row and column gap  props being used as you rightly pointed out they need to be included in the interface.